packages/components/psammead-story-promo/src/index.stories.jsx

```diff
@@ -45,6 +45,16 @@ const LiveComponent = ({ headline, service, dir }) => (
   </span>
 );
 
+/* eslint-disable-next-line react/prop-types */
+const HiddenText = ({ type, headline }) => (
+  /* eslint-disable-next-line jsx-a11y/aria-role */
+  <span role="text">
```

Comment:
why do we add "role=text" here?  just wondering, because I've seen warnings around pages about the use of unrecognised role values

https://www.w3.org/TR/wai-aria/#role_definitions

Reply:
`role="text"` is required to correct a text splitting bug on iOS VoiceOver. https://github.com/dequelabs/axe-core/issues/1597